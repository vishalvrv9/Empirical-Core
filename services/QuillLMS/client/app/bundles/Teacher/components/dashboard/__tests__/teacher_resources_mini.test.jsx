import React from 'react';
import { shallow } from 'enzyme';

import TeacherResourcesMini from '../teacher_resources_mini';

describe('TeacherResourcesMini component', () => {

  it('should render', () => {
    const wrapper = shallow(
      <TeacherResourcesMini />
    );
    expect(wrapper).toMatchSnapshot();
  });

});
