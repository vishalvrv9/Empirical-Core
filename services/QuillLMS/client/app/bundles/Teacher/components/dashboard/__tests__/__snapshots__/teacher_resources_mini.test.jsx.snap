// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeacherResourcesMini component should render 1`] = `
<div
  className="mini_container col-md-4 col-sm-5 text-center"
>
  <div
    className="mini_content"
  >
    <div
      className="resources-container"
      onClick={[Function]}
    >
      <h4>
        Teacher Center
      </h4>
      <img
        src="/teacher_resources_icons.png"
      />
      <p>
        Introduce fellow educators to Quill with presentations, guides, and videos.
      </p>
    </div>
  </div>
</div>
`;
